mission/edit_me/briefing/mission.html

```diff
@@ -0,0 +1,5 @@
+INSERT YOUR MISSION DESCRIPTION HERE,
+
+YOU CAN ADD AS MUCH AS YOU FEEL IS RELEVANT.
+
+AS A RULE OF THUMB, CONSIDER: WHO, WHAT (TASKS), WHERE, WHEN, AND WHY
```

Comment:
I reckon this needs colored headers or a title (aside from the built-in one) if we want it to follow the design system in SITUATION.